### PR TITLE
update computational backend links

### PR DESCRIPTION
**Pull request by danidata**
Update for #4776

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -38,16 +38,23 @@ If you are new to dplyr, the best place to start is the [data import chapter](ht
...
-  to access the excellent performance of [data.table](http://r-datatable.com/).
-  
-* For data in relational databases, [dbplyr](http://dbplyr.tidyverse.org/) 
-  will automatically translate your dplyr code in to SQL.
-  
-* For very large datasets stored in [Apache Spark](https://spark.apache.org), 
-  use [sparklyr](https://spark.rstudio.com).
+In addition to data frames/tibbles, dplyr makes working with other computational backends accesible and efficient. Below is a list of alternative backends :
+
+#### [dtplyr](https://dtplyr.tidyverse.org/)
```
Comment (hadley) [inline README.Rmd]: Can you keep this as a bulleted list please? I realise that the issue was confusing 😬

Comment (danidata) [reply README.Rmd]: Can do!

Comment (danidata) [reply README.Rmd]: Changed to bullet list! Also, I wasn't sure how descriptive I should be for each backend... I can make it more informative if desired

File: README.Rmd [hunk 2]
```diff
@@ -38,16 +38,19 @@ If you are new to dplyr, the best place to start is the [data import chapter](ht
...
-
-* For large, in-memory datasets, try [dtplyr](http://dtplyr.tidyverse.org/)
-  to access the excellent performance of [data.table](http://r-datatable.com/).
-  
-* For data in relational databases, [dbplyr](http://dbplyr.tidyverse.org/) 
-  will automatically translate your dplyr code in to SQL.
-  
-* For very large datasets stored in [Apache Spark](https://spark.apache.org), 
-  use [sparklyr](https://spark.rstudio.com).
+In addition to data frames/tibbles, dplyr makes working with other computational backends accesible and efficient. Below is a list of alternative backends:
```
Comment (romainfrancois) [inline README.Rmd]:
```suggestion
In addition to data frames/tibbles, dplyr makes working with other computational backends accessible and efficient. Below is a list of alternative backends:
```

File: README.Rmd [hunk 3]
```diff
@@ -38,16 +38,19 @@ If you are new to dplyr, the best place to start is the [data import chapter](ht
...
+
+- [dtplyr](https://dtplyr.tidyverse.org/): for large, in-memory datasets, try to access the excellent performance of [data.table](http://r-datatable.com/).
+
+- [dbplyr](http://dbplyr.tidyverse.org/): for data in relational databases, use
+[dbplyr](http://dbplyr.tidyverse.org/). dbplyr will automatically translate
```
Comment (romainfrancois) [inline README.Rmd]: Looks like this second link is redundant

File: README.Rmd [hunk 4]
```diff
@@ -38,16 +38,19 @@ If you are new to dplyr, the best place to start is the [data import chapter](ht
...
+    your dplyr code in to SQL.
+
+- [sparklyr](https://spark.rstudio.com): for very large datasets stored in [Apache Spark](https://spark.apache.org), check out [sparklyr](https://spark.rstudio.com).
```
Comment (romainfrancois) [inline README.Rmd]: Perhaps stop after the spark link, as there is already a `sparlyr` link in the bullet.

Comment (danidata) [reply README.Rmd]: Sure, I can see that. Make sense!

**Comment by hadley** [comment]
Thanks @danidata!

**Pull request merged**